vnext/src-win/Libraries/Text/Text.windows.js

```diff
@@ -142,16 +470,14 @@ const TextLegacy: React.AbstractComponent<
     }
 
     return {
-      disabled: !isPressable,
+      disabled: false,
```

Comment:
so it's always disabled?

Reply:
Looks like its always enabled.  - Looks like you cannot make text pressable + disabled. I assume this just aligns with upstream.

Reply:
yes this is also from upstream, disabled is defined above as normal on the other Text objects :)